sciter/value.py

```diff
@@ -690,11 +690,9 @@ def pack_to(scval, val):
     def pack_args(*args, **kwargs):
         """Pack arguments tuple as SCITER_VALUE array."""
         argc = len(args)
-        args_type = SCITER_VALUE * argc
```

Comment:
For a moment I thought that we could use `value * argc` to solve the leak but no - it's not a ctypes' structure. So, a wrapper indeed 👍